cmd/craned/app/manager.go

```diff
@@ -84,6 +93,60 @@ func Run(ctx context.Context, opts *options.Options) error {
 		return err
 	}
 
+	var dataSource providers.Interface
```

Comment:
would you please move it to  function initializationControllers

Reply:
because there has some other params, if so we should change the initializationControllers func params

Reply:
Add more params for initializationControllers is better